core/src/main/java/tech/tablesaw/io/ReadOptions.java

```diff
@@ -294,6 +301,15 @@ public Builder minimizeColumnSizes() {
       return this;
     }
 
+    /**
+     * @param b if true, reads strings that appear as percentages (eg. 10%) as numbers (0.1)
```

Comment:
If we don't make any changes to the logic, the code comment here is inadequate, since it doesn't explain what happens if there is no % on a given row. We should try to explain the API completely. 

Also, i didn't read this code carefully enough, but i assume its possible to apply the percentage option to a column of non-numeric data. We should probably state what error will occur if that happens.

Reply:
Hm, actually there's a potential problem here. The percentage flag is not set per column but per reader. So, all columns that end with % are treated as potential percentages. There's no way to say only a specific column is treated like this.

I wonder, how is this done for other column-level reader options?

Reply:
I don't think we have any options that are per-column

Reply:
> If we don't make any changes to the logic, the code comment here is inadequate, since it doesn't explain what happens if there is no % on a given row. 

Nothing should happen since it won't be detected as a percentage. If you mean that data is not clean and some values might have % and some not, then all bets are off anyhow.